app/models/solidus_friendly_promotions/friendly_promotion_discounter.rb

```diff
@@ -23,15 +23,22 @@ def call
         (line_item_discounts + shipment_discounts + shipping_rate_discounts).each do |item, chosen_discounts|
           item.current_discounts.concat(chosen_discounts)
         end
+        perform_actions(lane_promotions)
       end
 
       order
     end
 
     private
 
+    def perform_actions(promotions)
```

Comment:
Should this be `perform_automations` or something like that to differentiate that this is at the order level and not a regular action?